DigitalLearningSolutions.Web/Views/SuperAdmin/Faqs/_FaqCard.cshtml

```diff
@@ -0,0 +1,61 @@
+<div class="searchable-element nhsuk-expander-group word-break" id="@Model.FaqId-card">
+
+  <details class="nhsuk-details nhsuk-expander">
+    <div class="nhsuk-details__text">
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-one-quarter">
+          <div class="nhsuk-tag tag card-tag--grey">
+            @Model.TargetGroup
+          </div>
+        </div>
+        <div class="nhsuk-grid-column-one-quarter">
+          <div class="nhsuk-tag tag card-tag--grey">
+            @Model.Published
+          </div>
+        </div>
+      </div>
+    </div>
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text" id="@Model.FaqId-name">
+        <span class="searchable-element-title" name="name">@Model.QText</span>
+      </span>
+    </summary>
+
+    <div class="nhsuk-details__text">
+      <div class="nhsuk-u-margin-bottom-4">
+        <span class="searchable-element-content">@Html.Raw(Model.AHtml)</span>
+        <div class="nhsuk-grid-row">
+          <div class="nhsuk-grid-column-one-quarter">
+            <p class="nhsuk-u-font-weight-bold">Created</p>
+          </div>
+          <div class="nhsuk-grid-column-one-quarter">
+            <p>@Model.CreatedDate</p>
+          </div>
+        </div>
+        <div class="nhsuk-grid-row">
+          <div class="nhsuk-grid-column-one-quarter">
+            <p class="nhsuk-u-font-weight-bold">Weighting</p>
+          </div>
+          <div class="nhsuk-grid-column-one-quarter">
+            <p>@Model.Weighting</p>
+          </div>
+        </div>
```

Comment:
Think we normally put these in a summary list (without actions): https://service-manual.nhs.uk/design-system/components/summary-list